convert2rhel/unit_tests/conftest.py

```diff
@@ -68,3 +70,29 @@ def pkg_root(is_py2):
 @pytest.fixture(autouse=True)
 def setup_logger(tmpdir):
     initialize_logger(log_name="convert2rhel", log_dir=tmpdir)
+
+
+@pytest.fixture()
+def replace_data_dir_for_centos8(monkeypatch, pkg_root):
+    """Use our """
```

Comment:
Missing doc

Reply:
just removed it, and made private. it is not assumed to be used outside of the pretend_centos8